samples/attested_tls/README.md

```diff
@@ -88,3 +90,28 @@ On Windows after building the sample as described in the [README file](../README
 ```cmd
 .\server\host\tls_server_host .\server\enc\tls_server_enc.signed -port:12341 -server-in-loop
 ```
+### OpenSSL recommended configurations for TLS communication
+
+  Open Enclave's security guidance recommends to use only subset of cipher suites, elliptic curve algorithms and TLS protocols available in the OpenSSL library. The recommended algorithms and protocols are listed below. Enclave application developers can use [`initalize_ssl_context`](common/openssl_utility.cpp#L118) method to configure the SSL_CTX and limit the algorithms and protocols to below list. In Attested TLS sample application, server and client modules invoke the method `initalize_ssl_context` [here](server/enc/openssl_server.cpp#L147) and [here](client/enc/openssl_client.cpp#L200) respectively to account for the mentioned recommendations. It is strongly recommended that the application developers uses similar method to configure the SSL_CTX and limit the available ciphersuites and elliptic curve algorithms to below list.
```

Comment:
```suggestion
  Open Enclave's security guidance recommends to use only a subset of cipher suites in TLS 1.2/1.3 and elliptic curve algorithms available in the OpenSSL library. The [`initalize_ssl_context`](common/openssl_utility.cpp#L118) demonstrates how to configure both a [server](server/enc/openssl_server.cpp#L147) and a [client](client/enc/openssl_client.cpp#L200) with the recommended algorithms and protocols (see the full list below).
```

Reply:
I tried it and looks like existing mbedtls application runs with TLS 1.2. With the TLS 1.2 disabled, the communication between enclave mbedtls server and non_enclave openssl client failing with below error. 
`! mbedtls_ssl_handshake returned -0x7380
TLS server: server communication error -29568
TLS server: Last error was: -29568 - SSL - The server has no ciphersuites in common with the client`

Reply:
I guess you're replying to my question above. I see problem is that non_enc client is also expected to work with Mbed TLS server, correct? In that case, we can just leave as it is.

Reply:
Yes, I could not reply in there for some reason. Hence added comment here. Yes, we will leave as it is.